fallzahlen_kanton_total_csv/COVID19_Fallzahlen_Kanton_GE_total.csv

```diff
@@ -6,26 +6,26 @@ date,time,abbreviation_canton_and_fl,ncumul_tested,ncumul_conf,ncumul_hosp,ncumu
-2020-03-17,,GE,,495,75,19,54,,4,https://www.ge.ch/document/covid-19-situation-epidemiologique-geneve/telecharger,2,9,
-2020-03-18,,GE,,629,78,20,56,,5,https://www.ge.ch/document/covid-19-situation-epidemiologique-geneve/telecharger,2,17,
-2020-03-19,,GE,,826,92,19,68,,5,https://www.ge.ch/document/covid-19-situation-epidemiologique-geneve/telecharger,5,17,
-2020-03-20,,GE,,994,109,22,82,,8,https://www.ge.ch/document/covid-19-situation-epidemiologique-geneve/telecharger,5,21,
-2020-03-21,,GE,,1128,145,25,107,,10,https://www.ge.ch/document/covid-19-situation-epidemiologique-geneve/telecharger,13,24,
-2020-03-22,,GE,,1203,179,36,137,,9,https://www.ge.ch/document/covid-19-situation-epidemiologique-geneve/telecharger,6,36,4
+2020-03-03,,GE,,9,4,0,4,,,https://www.ge.ch/document/covid-19-situation-epidemiologique-geneve/telecharger,0,0,
+2020-03-04,,GE,,9,4,0,4,,,https://www.ge.ch/document/covid-19-situation-epidemiologique-geneve/telecharger,0,0,
+2020-03-05,,GE,,13,5,0,5,,,https://www.ge.ch/document/covid-19-situation-epidemiologique-geneve/telecharger,0,0,
+2020-03-06,,GE,,17,7,0,7,,,https://www.ge.ch/document/covid-19-situation-epidemiologique-geneve/telecharger,0,0,
+2020-03-07,,GE,,26,7,0,7,,,https://www.ge.ch/document/covid-19-situation-epidemiologique-geneve/telecharger,0,0,
+2020-03-08,,GE,,35,9,0,9,,,https://www.ge.ch/document/covid-19-situation-epidemiologique-geneve/telecharger,0,0,
+2020-03-09,,GE,,44,13,3,10,,1,https://www.ge.ch/document/covid-19-situation-epidemiologique-geneve/telecharger,0,0,
+2020-03-10,,GE,,68,18,4,14,,2,https://www.ge.ch/document/covid-19-situation-epidemiologique-geneve/telecharger,0,1,
+2020-03-11,,GE,,79,20,6,14,,2,https://www.ge.ch/document/covid-19-situation-epidemiologique-geneve/telecharger,0,1,
+2020-03-12,,GE,,108,23,6,17,,2,https://www.ge.ch/document/covid-19-situation-epidemiologique-geneve/telecharger,0,4,
+2020-03-13,,GE,,173,33,7,26,,2,https://www.ge.ch/document/covid-19-situation-epidemiologique-geneve/telecharger,0,4,
+2020-03-14,,GE,,282,43,8,33,,2,https://www.ge.ch/document/covid-19-situation-epidemiologique-geneve/telecharger,2,5,
+2020-03-15,,GE,,372,46,12,32,,4,https://www.ge.ch/document/covid-19-situation-epidemiologique-geneve/telecharger,2,5,
+2020-03-16,,GE,,472,66,10,52,,4,https://www.ge.ch/document/covid-19-situation-epidemiologique-geneve/telecharger,4,8,
+2020-03-17,,GE,,619,75,19,54,,4,https://www.ge.ch/document/covid-19-situation-epidemiologique-geneve/telecharger,2,9,
+2020-03-18,,GE,,751,78,20,56,,5,https://www.ge.ch/document/covid-19-situation-epidemiologique-geneve/telecharger,2,17,
+2020-03-19,,GE,,961,92,19,68,,5,https://www.ge.ch/document/covid-19-situation-epidemiologique-geneve/telecharger,5,17,
+2020-03-20,,GE,,1136,109,22,82,,8,https://www.ge.ch/document/covid-19-situation-epidemiologique-geneve/telecharger,5,21,
+2020-03-21,,GE,,1262,145,25,107,,10,https://www.ge.ch/document/covid-19-situation-epidemiologique-geneve/telecharger,13,24,
```

Comment:
Great! Just for curiostiy - have you stored the pdfs? How were you able to trace back these numbers?

Reply:
Have you seen : #199 @zukunft ?

Reply:
Now yes.

Reply:
Good point. But adding two dates would surely add a bit too much complexity. However, we might have to discuss how we can transparently declare such differences between cantons wdyt?

Reply:
If Geneva does really report with the "date of test taken", which I assume, the solution could be

1. scrape the complete data from the PDF on daily bases
2. save the complete data based on "date of test taken" on daily bases
3. build a script that calculates the "date of positive result" data based on the history of the "date of test taken" by comparing today's with yesterdays data

Should I create an issue for this and start with a prototype in python?